popSimManu.tex

```diff
@@ -132,24 +132,25 @@ \section*{Introduction}
 parameters and will also provide reasonable estimates of uncertainty of the
 estimated parameters. Indeed, simulations studies are a standard ingredient in
 most papers proposing new methods in population genetics. However, these
-simulation studies are often specifically devised to showcase the performance
-of the novel method and may not adequately compare performance to competing
-methods. Moreover researchers are generally forced to ``reinvent the wheel''
-with respect to simulations, and are forced to reimplement previously
-inferred scenarios independently. This can lead to errors being incorporated
-during the implementation of the simulations. Lastly, the degree to
-which different methods are compared can be quite variable.
+simulation studies often use simulations of the particular models for which the method was designed,
+which showcases the novel method rather than rigorously comparing performance to competing methods.
+Of course, testing a new method in the situation for which it was designed is an important step,
+but more rigorous testing is necessary for tools that are to be widely used in the field.
+% Moreover researchers are generally forced to reimplement previously
+% inferred scenarios independently, which makes good benchmarking
+% a major additional time investment, and can lead to errors.
+% Lastly, the degree to which different methods are compared can be quite variable.
```

Comment:
this overlapped a bit with the next paragraph

Reply:
also, wanted to not say that methods researchers were deliberately cherry-picking the best ones; would also like to acknowledge that simulating under the best-case model is a good thing to do.